### DISCUSSION TITLE
Channel Request： Fuji TV ONE and PIGOO(jp)

### DISCUSSION THREAD

**Discussion by okonomitarou** [iptv-org/community#1025] [Q&A]
These two streams cannot be found. 
Please help me!

Paste the link to your homepage. 
Fuji TV ONE →https://otn.fujitv.co.jp/
PIGOO→https://pigoo.jp/pigoohd/

By the way, the content is a Japanese entertainment channel.

Best Regard.

**Comment by BellezaEmporium** [comment]
As far as I know, those channels aren't made for online viewing (Over-the-Air only for specific Japanese regions ?), and are extremely rare to find otherwise (the only known way is people retransmitting those channels from Japan to another country and selling them to people).

![image](https://github.com/iptv-org/iptv-org.github.io/assets/30985701/3ff7af66-8031-49d9-a5a6-f6f97b9aad26)
Pigoo seems to be a paid channel too.

**Comment by okonomitarou** [comment]
Thanks for the message! 
Are both difficult to watch?

**Reply by BellezaEmporium** [reply]
Quite difficult to get outside of Japan indeed.

**Reply by okonomitarou** [reply]
Got it.　 
Give Up. 
thank you

**Reply by okonomitarou** [reply]
And,How do I close this discussion?
Regards

**Comment by BellezaEmporium** [comment]
Locked on OP's demand.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19902
https://github.com/iptv-org/iptv/issues/19903
